docker-compose.yml

```
@@ -322,6 +338,8 @@ services:
     restart: always
     container_name: elastic
     hostname: elastic
+    healthcheck:
+      test: wget -q --no-verbose --tries=1 --spider http://localhost:9200/_cat/health
```

```suggestion
      test: docker-healthcheck
```
See https://github.com/blacktop/docker-elasticsearch-alpine/blob/master/7.6/docker-healthcheck